presto-accumulo/src/test/java/com/facebook/presto/accumulo/TestAccumuloDistributedQueries.java

```diff
@@ -0,0 +1,503 @@
+                multipleRowsErrorMsg);
+        assertQueryFails("SELECT orderkey, totalprice FROM orders ORDER BY (VALUES 1, 2)",
+                multipleRowsErrorMsg);
+
+        // exposes a bug in optimize hash generation because EnforceSingleNode does not
+        // support more than one column from the underlying query
+        assertQuery("SELECT custkey, (SELECT DISTINCT custkey FROM orders ORDER BY custkey LIMIT 1) FROM orders");
+    }
+
+    @Override
+    public void testShowColumns()
+            throws Exception
+    {
+        // Override base class because table descriptions for Accumulo connector include comments
+        MaterializedResult actual = computeActual("SHOW COLUMNS FROM orders");
+
+        assertEquals("orderkey", actual.getMaterializedRows().get(0).getField(0));
+        assertEquals("bigint", actual.getMaterializedRows().get(0).getField(1));
+        assertEquals("custkey", actual.getMaterializedRows().get(1).getField(0));
+        assertEquals("bigint", actual.getMaterializedRows().get(1).getField(1));
+        assertEquals("orderstatus", actual.getMaterializedRows().get(2).getField(0));
+        assertEquals("varchar", actual.getMaterializedRows().get(2).getField(1));
+        assertEquals("totalprice", actual.getMaterializedRows().get(3).getField(0));
+        assertEquals("double", actual.getMaterializedRows().get(3).getField(1));
+        assertEquals("orderdate", actual.getMaterializedRows().get(4).getField(0));
+        assertEquals("date", actual.getMaterializedRows().get(4).getField(1));
+        assertEquals("orderpriority", actual.getMaterializedRows().get(5).getField(0));
+        assertEquals("varchar", actual.getMaterializedRows().get(5).getField(1));
+        assertEquals("clerk", actual.getMaterializedRows().get(6).getField(0));
+        assertEquals("varchar", actual.getMaterializedRows().get(6).getField(1));
+        assertEquals("shippriority", actual.getMaterializedRows().get(7).getField(0));
+        assertEquals("integer", actual.getMaterializedRows().get(7).getField(1));
+        assertEquals("comment", actual.getMaterializedRows().get(8).getField(0));
+        assertEquals("varchar", actual.getMaterializedRows().get(8).getField(1));
+    }
+
+    // Copied from abstract base class
+    private void assertCreateTableAsSelect(String table, @Language("SQL") String query, @Language("SQL") String rowCountQuery)
+            throws Exception
+    {
+        assertCreateTableAsSelect(getSession(), table, query, query, rowCountQuery);
+    }
+
+    // Copied from abstract base class
+    private void assertCreateTableAsSelect(String table, @Language("SQL") String query, @Language("SQL") String expectedQuery, @Language("SQL") String rowCountQuery)
+            throws Exception
+    {
+        assertCreateTableAsSelect(getSession(), table, query, expectedQuery, rowCountQuery);
+    }
+
+    // Copied from abstract base class
```

Comment:
Then change them in base class to be protected. This would at least make a clear indication for others working with base class that someone is using it.